examples/getting-started-movielens/04-Triton-Inference-with-TF.ipynb

```diff
@@ -35,26 +35,7 @@
     "\n",
```

Comment:
<p>you can write down as <code> We need to start the Triton Inference Server first. We can do that with the following command.  You need to provide correct path for the&nbsp;models&nbsp;directory. Note that  since we add --model-control-mode=explicit flag the models wont be loaded yet, we will load our model below.</code> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>change made</p>